### ISSUE TITLE
Implement square scaling on the graphs and instrument window

### ISSUE THREAD

**Issue by mantid-builder** [mantidproject/issue-migration-testing#272]
Squares% goes 1, 4, 9, 16, 25, 35 .. % of maximum

**Comment by mantid-builder** [comment]
This issue was originally trac ticket [771](http://trac.mantidproject.org/mantid/ticket/771)

**Comment by mantid-builder** [comment]
@NickDraper : Moved as part of iteration 18 end

**Comment by mantid-builder** [comment]
@NickDraper : Moved as part of the end of Iteration 19

**Comment by mantid-builder** [comment]
@NickDraper : Bulk move of tickets at the end of iteration 27

**Comment by mantid-builder** [comment]
@NickDraper : Bulk move of tickets at the end of iteration 28

**Comment by mantid-builder** [comment]
@NickDraper : Accepted and assigned tickets moved at iteration 29 code freeze

**Comment by mantid-builder** [comment]
@NickDraper : Bulk move of tickets to iteration 31 at the iteration 30 code freeze

**Comment by mantid-builder** [comment]
@NickDraper : Moved to iteration 33 at iteration 32 code freeze

**Comment by mantid-builder** [comment]
@NickDraper : Moved at end of release 2.1

**Comment by mantid-builder** [comment]
@NickDraper : Moved at the end of release 2.2

**Comment by mantid-builder** [comment]
@NickDraper : Moved to milestone 2.4

**Comment by mantid-builder** [comment]
@NickDraper : Moved at the code freeze for release 2.4

**Comment by mantid-builder** [comment]
@NickDraper : Moved to r2.6 at the end of r2.5

**Comment by mantid-builder** [comment]
@NickDraper : I've added a proof of concept to a branch for this and published it to github.  This shows that we can inject our own scale engine and influence the qwt usage.

In this proof of concept I have added a QwtSquareScaleEngine into the MantidWidgets, and used it in the instrument display code as a replacement for the linear scale engine.

Note: This should not be taken forward as is as it is a hacked scale so I could see a difference and not useful in itself.
However the class structure could be used if the code is replaced with something more useful.

Note:The changes in ColorMapWidget will have to be undone.
